workflow.rst

```diff
@@ -305,7 +305,7 @@ order:
 
 .. note::
 
-    The leaving and entering events are triggered even for transitions that stay
+    The leaving and enter events are triggered even for transitions that stay
```

Comment:
i would reference the real event "string" (or fqdn?) instead (https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Workflow/WorkflowEvents.php#L18)

also why did you target 4.2?

Reply:
Just a note regarding the branch: this morning, I could not find this doc before 4.2. Now I can see it. Maybe it was moved or something? Really I don't know.